### PR TITLE
Ensured that Script window short cut keys work correctly

### PR DESCRIPTION
**Pull request by Patowhiz**
Partly fixes issue #8301.
@africanmathsinitiative/developers this is ready for review.

@lloyddewit as I understand it, the previous code replaced here, attempted to combine the top visible buttons with the context menu options.
My approach leans towards attempting to treat them differently in implementation. Unlike the top buttons, which are always visible, right click menu options are only visible when the user rights clicks. So I'm now disabling them during opening.
If a user attempts to use short cut keys, then the individual events call functions that give sensible feedback to the users.
This is what was previous there.

Thanks

### CODE REVIEW

**Review by rdstern — DISMISSED** [review]
Another "first-time round".  Seems to work fine.  Maybe I am not critical enough.

File: instat/ucrScript.vb [hunk 1]
```diff
@@ -236,40 +236,42 @@ Public Class ucrScript
...
     Private Sub EnableRunButtons(bEnable As Boolean)
         cmdRunLineSelection.Enabled = bEnable
         cmdRunAll.Enabled = bEnable
     End Sub
 
+    ''' <summary>
+    ''' Enables or disables all right click menu options
+    ''' </summary>
+    ''' <param name="bEnable">If true, enables all right click options,false otherwise</param>
+    Private Sub EnableRightClickMenuOptions(bEnable)
```
Comment (lloyddewit) [inline instat/ucrScript.vb:260]:
```suggestion
    Private Sub EnableRightClickMenuOptions(bEnable As Boolean)
```

File: instat/ucrScript.vb [hunk 2]
```diff
@@ -236,40 +236,42 @@ Public Class ucrScript
     Private Sub EnableDisableButtons()
 
         Dim bIsLogTab As Boolean = TabControl.SelectedIndex = iTabIndexLog
-
-        mnuUndo.Enabled = clsScriptActive.CanUndo AndAlso Not bIslogTab
-        mnuRedo.Enabled = clsScriptActive.CanRedo AndAlso Not bIslogTab
-
         Dim bScriptselected = clsScriptActive.SelectedText.Length > 0
```
Comment (lloyddewit) [inline instat/ucrScript.vb:239]:
```suggestion
```

**Review by lloyddewit — CHANGES_REQUESTED** [review]
@Patowhiz Thank you, this is a good approach.
Just 2 small suggestions

**Review by rdstern — APPROVED** [review]
Nice!

**Review by lloyddewit — APPROVED** [review]

**Pull request merged**